### DISCUSSION TITLE
wired problem with mutiple conftest files

### DISCUSSION THREAD

**Discussion by uestcheng** [pytest-dev/pytest#11796] [General]
Hi guys, my structure is as 
```
---core function (including the API object)
---test_case 
--- conftest.py
```
And I want to set the module level fixture for test_case, and my project's structure change to 
```
--- core function
--- test_case
            |____ sub_dir1
            |____ conftest.py # new conftest file for sub_dir1
---conftest # the older one
```
And there is an ```ImporError``` appears in the new conftest. I also tried append sys path, insert sys path. Does not work. Could you guys please give me another solution?

**Comment by The-Compiler** [comment]
Can you show us the exact error message (and ideally `conftest.py` contents) instead of vaguely describing it? Not much that can be said if all you give use is "there was some sort of `ImportError`"...

**Comment by uestcheng** [comment]
@The-Compiler , Thanks very much for this reply, My original code in new conftest.py 👇 
```Python
from core.api.gitlab.Namespace import Namespace
```
This will cause the ImportError
```bash
ImportError while loading conftest '/Users/tom/code/python3/myapi/test_case/gitlab/conftest.py'.
test_case/gitlab/conftest.py:5: in <module>
    from core.api.gitlab.namespace import Namespace
```

and I change this code to 👇 :
```bash
from core import api
```
it works, so I think the root cause maybe the relative import.
